opensearch/src/main/java/org/opensearch/sql/opensearch/data/type/OpenSearchIpType.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+package org.opensearch.sql.opensearch.data.type;
+
+import static org.opensearch.sql.data.type.ExprCoreType.UNKNOWN;
+
+import lombok.EqualsAndHashCode;
+
+@EqualsAndHashCode(callSuper = false)
```

Comment:
Can you add a javadoc with a link to the opensearch doc?

Reply:
Added in ae11c22.